metadata-ingestion/src/datahub/ingestion/source/salesforce.py

```diff
@@ -537,20 +537,24 @@ def get_profile_workunit(
 
     # Here field description is created from label, description and inlineHelpText
     def _get_field_description(self, field: dict, customField: dict) -> str:
-        desc = (
-            "\\" + field["Label"] if field["Label"].startswith("#") else field["Label"]
-        )
-
-        for key in ["FieldDefinition", "InlineHelpText"]:
-            text: Optional[str] = ""
-            if isinstance(field.get(key), dict):
-                text = field[key].get("Description")
-            else:
-                text = field.get(key)
-
-            if text:
-                prefix = "\\" if text.startswith("#") else ""
-                desc += f"\n\n{prefix}{text}"
+        if "Label" not in field or field["Label"] is None:
+            desc = ""
+        elif field["Label"].startswith("#"):
+            desc = "\\" + field["Label"]
+        else:
+            desc = field["Label"]
+
+        text = field.get("FieldDefinition", {}).get("Description", {})
```

Comment:
```
        text = field.get("FieldDefinition", {}).get("Description", None)
```

Reply:
yes, noticed that... some how the commit got messed-up.. correcting now